test/lib/jdk/test/lib/thread/ProcessThread.java

```diff
@@ -150,16 +150,23 @@ public ProcessRunnable(ProcessBuilder pb, String name, Predicate<String> waitfor
          */
         @Override
         public void xrun() throws Throwable {
-            this.process = ProcessTools.startProcess(name, processBuilder, waitfor);
-            // Release when process is started
-            latch.countDown();
+            String name = Thread.currentThread().getName();
```

Comment:
The original code passes the name of the `ProcessRunnable` to `startProcess`, not the name of the current thread. It is not obvious/apparent that they are the same. But if they are then it is cheaper to use `name` than do a dynamic query on the current thread.

Reply:
Thanks! Fixed, it is a mistake.